ocw/lib/eks.py

```diff
@@ -86,15 +86,15 @@ def kubectl_client(self, region: str, cluster_name: str):
     def all_clusters(self) -> dict:
         clusters = {}
         for region in EKS.__cluster_regions:
-            self.log_dbg("Checking clusters in {}", region)
+            self.log_dbg(f"Checking clusters in {region}")
```

Comment:
I would like to use same style everywhere , if you want to switch to f-strings please prepare commit which do it everywhere not just in few files

Reply:
> I would like to use same style everywhere , if you want to switch to f-strings please prepare commit which do it everywhere not just in few files

Will do it in this PR then.

Reply:
> I would like to use same style everywhere , if you want to switch to f-strings please prepare commit which do it everywhere not just in few files

There are far too many lines to change for me to make that change right now.  In some instances the line will exceed the line limit of 130 chars.  So either we do it one module at a time or I rollback the f-string changes in this file.

Reply:
> I would like to use same style everywhere , if you want to switch to f-strings please prepare commit which do it everywhere not just in few files

These Kubernetes scripts already use f-strings:
- [https://github.com/SUSE/pcw/blob/master/ocw/lib/aks.py](ocw/lib/aks.py)
- [https://github.com/SUSE/pcw/blob/master/ocw/lib/gke.py](ocw/lib/gke.py)
- [https://github.com/SUSE/pcw/blob/master/ocw/lib/k8s.py](ocw/lib/k8s.py)

GCE & Openstack also use them.  I promise to change EC2 & Azure when I touch them.

Reply:
ok